client/my-sites/sharing/connections/service.jsx

```diff
@@ -434,6 +443,15 @@ export class SharingService extends Component {
 						numberOfConnections={ this.getConnections().length }
 					/>
 				</div>
+				{ 'linkedin' === this.props.service.ID &&
+					some( connections, { status: 'must_reauth' } ) && (
```

Comment:
It seems like we'd want to show this message when the `linkedin` service has status `must_reauth`, but we'd show it if any service has that status. Is that correct?

Reply:
I agree with Jon here. I think we should make sure that we check against the `linkedin` connections.

Also, note that there might be multiple LinkedIn connections, so we might end up displaying this more than once, which I doubt we'd want to do.

Reply:
I see what you mean since I was confused like that at the beginning. The thing is that this is not a list of connectionS, but a list with a single connection.